.github/workflows/test.yml

```
@@ -34,40 +34,41 @@ jobs:
         # - PHP 8.4 needs PHPCS 3.11.0+ to run without errors (though the errors don't affect this package).
         #
         # The matrix is set up so as not to duplicate the builds which are run for code coverage.
-        php: ['5.5', '5.6', '7.0', '7.1', '7.2', '7.3']
-        phpcs_version: ['3.5.6', 'dev-master']
+        php: ["5.5", "5.6", "7.0", "7.1", "7.2", "7.3"]
+        phpcs_version: ["3.5.6", "4.0.0", "dev-4.x"]
```

```suggestion
        phpcs_version: ["3.5.6", "4.0.0", "4.x-dev"]
```

(and the same in the other builds in `include` below)